### PR TITLE
GEODE-3788: add alter async-event-queue command and tests

### PR DESCRIPTION
**Pull request by jinmeiliao**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [x] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [x] Is your initial contribution a single, squashed commit?

- [x] Does `gradlew build` run cleanly?

- [x] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/json/GfJsonObject.java [hunk 1]
```diff
@@ -305,6 +305,11 @@ public boolean has(String key) {
     return jsonObject.keys();
   }
 
+  /**
+   * this returns the column size
+   *
+   * @return
```
Comment (PurelyApplied) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/json/GfJsonObject.java]: The description should be in-line with `@return` for the javadoc, i.e.,

```
  /**
   * @return the column size of the jsonObject
   */
  public int size() {
    return jsonObject.length();
  }
```

**Comment by jinmeiliao** [comment]
precheckin green

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/AlterAsyncEventQueueCommand.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  static final String ID = "id";
+  static final String BATCH_SIZE = "batch-size";
+  static final String BATCH_TIME_INTERVAL = "batch-time-interval";
+  static final String MAX_QUEUE_MEMORY = "max-queue-memory";
+  static final String MAXIMUM_QUEUE_MEMORY = "maximum-queue-memory";
+
+  static final String COMMAND_HELP =
+      "alter attributes of async-event-queue, needs rolling restart for new attributes to take effect. ";
+  static final String ID_HELP = "Id of the async event queue to be changed.";
+  static final String BATCH_SIZE_HELP = CliStrings.CREATE_ASYNC_EVENT_QUEUE__BATCH_SIZE__HELP;
```
Comment (pdxrunner) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/AlterAsyncEventQueueCommand.java]: For consistency, import this string from CliStrings as you did for the following help strings.

**Comment by jinmeiliao** [comment]
good catch, Jens! I did miss putting the updated config back to the region.

**Comment by jinmeiliao** [comment]
merged.

**Pull request closed without merging**